src/services/database/svc/mysql-uow.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Inject, Injectable } from 'injection-js';
 import { MysqlError, PoolConnection } from 'mysql';
 import { defer, from, Observable } from 'rxjs';
-import { catchError, mergeMap } from 'rxjs/operators';
+import { catchError, map, mergeMap } from 'rxjs/operators';
```

Comment:
Just curious, why is map being imported here? I don't see it being used anywhere in this file.

Reply:
Bc I forgot to remove it. Good catcg

Reply:
has this been fixed?

Reply:
Fixing now.

Reply:
Fixed